source/views/sis/index.js

```diff
@@ -4,13 +4,15 @@ import {TabNavigator} from '../components/tabbed-view'
 
 import BalancesView from './balances'
 import StudentWorkView from './student-work'
+import CourseSearchView from './course-search'
 import TESView from './tes'
 
 export default TabNavigator(
 	{
 		BalancesView: {screen: BalancesView},
 		TESView: {screen: TESView},
 		StudentWorkView: {screen: StudentWorkView},
+		CourseSearchView: {screen: CourseSearchView},
```

Comment:
I feel like this should be the second tab… seeing as it's the only other piece of this that actually comes from SIS? Thoughts?